frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx

```diff
@@ -58,23 +68,49 @@ const ProjectTableRow: React.FC<ProjectTableRowProps> = ({
         )}
       </Td>
       <Td isActionCell>
-        <ActionsColumn
-          items={[
-            {
-              title: 'Edit project',
-              isDisabled: isRefreshing,
-              onClick: () => {
-                setEditData(project);
+        {allowCreate ? (
```

Comment:
We could apply the boolean flag to the dropdown item only and spread it, so we don't need to create duplicated codes here. Here is the solution:
```
<Td isActionCell>
  <ActionsColumn
    items={[
      {
        title: 'Edit project',
        isDisabled: isRefreshing,
        onClick: () => {
          setEditData(project);
        },
      },
      {
        title: 'Delete project',
        onClick: () => {
          setDeleteData(project);
        },
      },
      ...(allowCreate
        ? [
             {
               title: 'Edit permissions',
               onClick: () => {
                 navigate(`/projects/${project.metadata.name}`, { state: 'Permissions' });
               },
             },
           ]
        : []),
    ]}
    And......
```